test/libsolidity/syntaxTests/dataLocations/function_argument_location_specifier_test.sol

```diff
@@ -0,0 +1,8 @@
+contract test {
+    function fa(bytes) public;
+    function fb(bytes memory) public;
+    function fc(bytes storage) public;
+    function fd(bytes calldata) public;
+}
+// ----
+// TypeError: (101-106): Location has to be memory for publicly visible functions (remove the "storage" or "calldata" keyword).
```

Comment:
Why is there only one error? There should be two, one for `fc`, one for `fd`.

Reply:
Ah it is a fatal one! In that case, please change this test case to have exactly one function.